actor/builtin/miner/miner.go

```diff
@@ -58,6 +57,8 @@ const (
 	ErrAskNotFound = 40
 	// ErrInvalidSealProof signals that the passed in seal proof was invalid.
 	ErrInvalidSealProof = 41
+	// ErrGetProofsModeFailed indicates the call to get the sector store type failed.
```

Comment:
suggestion: "ErrGetProofsModeFailed indicates the call to get the proofs mode failed."